tests/test-cases/Group24-Multi-VCH/24-02-Multi-VCH-Delete.robot

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 24-02 - Verify vic-machine delete only removes own cVMs
+Resource  ../../resources/Util.robot
+Suite Teardown  Cleanup VCHs
+
+*** Keywords ***
+Cleanup VCHs
+    Run Keyword And Ignore Error  Cleanup VIC Appliance On Test Server
+    Set Environment Variable  VCH-NAME  ${old-vm}
+    Set Environment Variable  BRIDGE_NETWORK  ${old-vch-bridge}
+    Set Environment Variable  VCH-PARAMS  ${old-vch-params}
+    Run Keyword And Continue On Failure  Cleanup VIC Appliance On Test Server
+
+*** Test Cases ***
+VCH delete only removes its own containers
+    ${c1}=  Evaluate  'cvm-vch1-' + str(random.randint(1000,9999))  modules=random
+
+    Install VIC Appliance To Test Server
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${busybox}
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${rc}=  Run And Return Rc  docker %{VCH-PARAMS} create --name ${c1} ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+
+    Set Suite Variable  ${old-vm}  %{VCH-NAME}
```

Comment:
nit: why not `old-vch` instead of `old-vm`

Reply:
Sure, will fix.